utils/fo-cleanold

```diff
@@ -244,8 +241,7 @@ crontab -r -u fossy
 if [ $RMREPO ]; then
    echo "*** Deleting repository ***"
    if [ -d /srv/fossology/repository ]; then
-      rm -rf /srv/fossology
-      if [ $? != 0 ]; then
+      if rm -rf /srv/fossology; then
```

Comment:
There is a logic inversion here, we should have: ` if ! rm -rf /srv/fossology; then`